src/middleware/translation.ts

```diff
@@ -23,4 +23,6 @@ i18next
         debug: false
     });
 
-export { i18next, i18nextMiddleware, ENGLISH, WELSH };
+const t = i18next.t;
+
+export { t, i18next, i18nextMiddleware, ENGLISH, WELSH };
```

Comment:
I went with `t` because that's what is documented against the library.  So it was more consistency... t in our context is always translate and a reference to `i18next` library.